CharlieBackend.Api/Controllers/HomeworkStudentController.cs

```diff
@@ -0,0 +1,92 @@
+﻿using CharlieBackend.Core.DTO.HomeworkStudent;
+using CharlieBackend.Business.Services.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using CharlieBackend.Core;
+
+namespace CharlieBackend.Api.Controllers
+{
+    /// <summary>
+    /// Controller to make operations with homework from student
+    /// </summary>
+    [Route("api/homeworkstudent")]
+    [ApiController]
+
+    public class HomeworkStudentController : ControllerBase
+    {
+
+        private readonly IHomeworkStudentService _homeworkStudentService;
+
+        /// <summary>
+        /// HomeworkStudent controllers constructor
+        /// </summary>
+        public HomeworkStudentController(IHomeworkStudentService homeworkStudentService)
+        {
+            _homeworkStudentService = homeworkStudentService;
+        }
+
+        /// <summary>
+        /// Adds homework from student
+        /// </summary>
+        /// <param name="request">
+        /// 1.HomeworkId -  it is id homework which one student done 
+        /// 2.HomeworkTest - here Student can write his Homework or something else
```

Comment:
```suggestion
        /// 2.HomeworkText - here Student can write his Homework or something else
```
what do you mean by *something else* ? :D

Reply:
@gingerbread-monster  i dont know